aspnetcore/blazor/components/integration.md

```diff
@@ -18,13 +18,210 @@ uid: blazor/components/integration
 
 This article explains Razor component integration scenarios for ASP.NET Core apps, including prerendering of Razor components on the server.
```

Comment:
We have a [separate article now on prerendering](https://learn.microsoft.com/aspnet/core/blazor/components/prerender). It seems like we should keep this article focused on component integration.

```suggestion
This article explains Razor component integration scenarios for ASP.NET Core apps.
```